### PR TITLE
Implement V128 print

### PR DESCRIPTION
**Pull request by GorogPeter**

### CODE REVIEW

File: src/runtime/Value.h [hunk 1]
```diff
@@ -53,6 +53,21 @@ struct Vec128 {
         return memcmp(m_data, src.m_data, 16) == 0;
     }
 
+    operator std::string() const
+    {
+        std::string result = "";
+        for (int i = 0; i < 16; i++) {
+            char left = (m_data[i] & 0b11110000) >> 4;
```
Comment (zherczeg) [inline src/runtime/Value.h:60]: we could use hex numbers, `0xf0` here

**Comment by GorogPeter** [comment]
Now, it looks like this:
```
invoke min-abs(0x00:00:90:bf:00:00:90:bf:00:00:90:bf:00:00:90:bf, 0x00:00:00:3e:00:00:00:3e:00:00:00:3e:00:00:00:3e) expect value(0x00:00:00:3e:00:00:00:3e:00:00:00:3e:00:00:00:3e) (line: 2405) : OK
Assertion failed at 2410: min-abs(0x00:00:90:bf:00:00:90:bf:00:00:90:bf:00:00:90:bf, 0x00:00:00:3e:00:00:00:3e:00:f0:48:45:00:00:00:3e) expected 0x00:00:00:3e:00:00:00:3e:00:00:00:3e:00:00:00:3e, but got 0x00:00:00:3e:00:00:00:3e:00:00:90:3f:00:00:00:3e
```

File: src/runtime/Value.h [hunk 2]
```diff
@@ -53,6 +53,21 @@ struct Vec128 {
         return memcmp(m_data, src.m_data, 16) == 0;
     }
 
+    operator std::string() const
+    {
+        std::string result = "0x";
+        for (int i = 0; i < 16; i++) {
```
[the remark this answers is not in the document]
Comment (GorogPeter) [reply src/runtime/Value.h:59]: Good idea, thanks!

[the remark this answers is not in the document]
Comment (zherczeg) [reply src/runtime/Value.h:59]: Just create a loop which generates 'hh:' pattern, and convert the last `':'` to `'\0'`.

**Comment by GorogPeter** [comment]
@zherczeg , your idea inspired me. Hence it's a fixed-length char array, it can be initialised with the "0x", the ":" and the "\0". Thus it's a wee-bit faster.

File: src/runtime/Value.h [hunk 3]
```diff
@@ -53,6 +53,18 @@ struct Vec128 {
...
+    operator std::string() const
+    {
+        char result[] = "0x00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00";
+        for (int i = 0; i < 16; i++) {
+            char left = (m_data[i] & 0xf0) >> 4;
+            char right = m_data[i] & 0x0f;
+            result[2 + 3 * i] = (left < 10) ? ('0' + left) : ('a' + (left - 10));
+            result[3 + 3 * i] = (right < 10) ? ('0' + right) : ('a' + (right - 10));
+        }
+        return std::string(result);
```
Comment (zherczeg) [inline src/runtime/Value.h:65]: Something like:

```
char result[16 * 3];
char* ptr = result;

loop {
  ptr[0] = first hex
  ptr[1] = second hex
  ptr[2] = ':'
  ptr += 3;
}

ptr[-1] = '\0';
```

**Review by zherczeg — APPROVED** [review]
LGTM

**Pull request merged**